frame/support/test/tests/benchmark_ui/bad_param_name.stderr

```diff
@@ -0,0 +1,7 @@
+error[E0599]: no variant or associated item named `winton` found for enum `BenchmarkParameter` in the current scope
+ --> tests/benchmark_ui/bad_param_name.rs:5:1
+  |
+5 | #[benchmarks]
+  | ^^^^^^^^^^^^^ variant or associated item not found in `BenchmarkParameter`
```

Comment:
Shouldn't this point to `winton`?

Reply:
Also, this error message is actually pretty cryptic, I wouldn't have known to change `winton` to `x` or `y` based on what it's telling me.

Reply:
same issue here with cloning a span I think -- apparently that just breaks and it goes to the enclosing element. I'll tweak the wording as well

Reply:
actually statically I don't think I can unless I hard-code the values of that enum. The reason the span is going to the `#[benchmarks]` is because this is an enum / impl error not something that I'm throwing

Reply:
here we go this is better:

```
error: Benchmark parameter names must consist of a single lowercase letter (a-z) and no other characters.
  --> tests/benchmark_ui/bad_param_name.rs:10:11
   |
10 |     fn bench(winton: Linear<1, 2>) {
   |              ^^^^^^
```